news/pr3735.feature.rst

```diff
@@ -0,0 +1,2 @@
+Improved compiled libraries dependency tree build time using multiple
```

Comment:
Pardon? Understanding "compiled libraries dependency tree build time" required some thinking. Please use a simpler language and prepositions (e.g. of, for. Thanks.